evaluation.md

```diff
@@ -5,15 +5,39 @@
 _Describe each change from your ideal syntax to the syntax you implemented, and
 describe_ why _you made the change._
 
+   * Needed to add () to the stuff before and after the ->: Needed so Scala could parse the syntax correctly
+   * Needed to replace : with +: Scala didn't like using : since it is widely used in the language already
+   * Needed to replace => with ->: Scala didn't like using => since it is widely used in the language already.
+   * Originally wanted to use | and ->: Needed to switch to + and -> so order of operations was the same
+
+
 **On a scale of 1–10 (where 10 is "a lot"), how much did you have to change your syntax?**
 
+3 -- We needed to add a few extra () and "" to our code to make it work. Additionally, we couldn't use ':' and '=>'
+as operators as originally intended because these are a part of the scala language. We then attempted to use "|"
+and "->", but this didn't work since Scala has predefined operator order of operations. We were able to get "+" and "->"
+to work since "+" and "-" have the same precedence since an operator's priority is based on the first character.
+
+
```

Comment:
Seems like almost everyone had to add parentheses. Use of + maaaaybe is confusing, but I think this is far enough from other standard uses for + that it's probably fine.